src/Controls/src/Build.Tasks/Controls.Build.Tasks.csproj

```diff
@@ -80,5 +107,5 @@
 			<_CopyItems Include="$(TargetDir)*.dll" />
 		</ItemGroup>
 		<Copy SourceFiles="@(_CopyItems)" DestinationFolder="..\..\..\..\.nuspec\$(TargetFramework)\" ContinueOnError="true" Retries="0" />
```

Comment:
We need the copy or it won't build in the IDE from a clean checkout as it wont exist. But, what I will say is that copying to .nuspec is not the best, maybe we should copy to a different location and also include the targets and props there.

Reply:
@StephaneDelcroix  ?

Reply:
I fixed this. The signing stuff still needs to be done again.